hatch_build.py

```diff
@@ -414,6 +414,7 @@
     # Blinker use for signals in Flask, this is an optional dependency in Flask 2.2 and lower.
     # In Flask 2.3 it becomes a mandatory dependency, and flask signals are always available.
     "blinker>=1.6.2",
+    "cloudpickle",
```

Comment:
We should not make cloudpickle required dependency - we know dill is needed for executor config, but we should rather leave cloudpickle as optional dependency only.

Reply:
`CORE_EXTRAS` is where we shoulkd add add it (same as virtualenv).

Reply:
Fixed that. Thanks for explaining!